arangod/V8Server/v8-actions.cpp

```diff
@@ -1738,8 +1738,16 @@ static void JS_DebugClearFailAt(v8::FunctionCallbackInfo<v8::Value> const& args)
     }
     std::string dbname(v8g->_vocbase->name());
 
+    std::string pathUrl;
+    if (args.Length() == 0) {
+      pathUrl = "_admin/debug/failat";
+    } else {
+      std::string const point = TRI_ObjectToString(isolate, args[0]);
+      pathUrl = "_admin/debug/failat/" + point;
+    }
+
     auto res =
-        clusterSendToAllServers(isolate, dbname, "_admin/debug/failat",
+        clusterSendToAllServers(isolate, dbname, pathUrl,
                                 arangodb::rest::RequestType::DELETE_REQ, "");
```

Comment:
Ok, now `JS_DebugClearFailAt` can do what `JS_DebugRemoveFailAt` can already do:
https://github.com/arangodb/arangodb/pull/13939/files?diff=unified&w=1#diff-284d4d1b634fa1caa50716432cab64ca7e6c6679df6b2c35db7a30103615604eR1677

Previously we had 2 distinct methods:
* "clear": unconditionally remove all failure points
* "remove": remove a single failure point by name
Now "clear" supports both modes, which is ok for me. Just mentioning it here.

Reply:
That is a good catch.
On client there was only clearFailAt(?name) which could take an optional failurePoint Name. The server side did not.
The server offers removeFailAt(name) and clearFailAt().
I have aliased removeFailAt and clearFailAt on client, and will remove the special JS_ implementation here.